### PR TITLE
add sports places & sport vision api

### PR DESCRIPTION
**Pull request by heri**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -992,6 +992,8 @@ API | Description | Auth | HTTPS | CORS |
 | [NBA Stats](https://any-api.com/nba_com/nba_com/docs/API_Description) | Current and historical NBA Statistics | No | Yes | Unknown |
 | [NHL Records and Stats](https://gitlab.com/dword4/nhlapi) | NHL historical data and statistics | No | Yes | Unknown |
 | [Sport List & Data](https://developers.decathlon.com/products/sports) | List of and resources related to sports | No | Yes | Yes |
+| [Sport Places](https://developers.decathlon.com/products/sport-places) | Crowd-source sports places around the world | No | Yes | No |
+| [Sport Vision](https://developers.decathlon.com/products/sport-vision) | Identify sport, brands and gear in an image. Also does image sports captioning | Yes | Yes | Yes |
```
Comment (matheusfelipeog) [inline README.md:996]: Fix this:

auth value is not enclosed with \`backticks\`
Yes is not a valid Auth option

Currently, the only accepted inputs for the Auth field are as follows:

`OAuth` - the API supports OAuth
`apiKey` - the API uses a private key string/token for authentication - try and use the correct parameter
`X-Mashape-Key` - the name of the header which may need to be sent
`User-Agent` - the name of the header to be sent with requests to the API
No - the API requires no authentication to run

```suggestion
| [Sport Vision](https://developers.decathlon.com/products/sport-vision) | Identify sport, brands and gear in an image. Also does image sports captioning | `apiKey` | Yes | Yes |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @heri :)

Can you resolve this review, please?

File: README.md (hunk 1 above)
Comment (heri) [reply README.md]: just corrected, thanks @matheusfelipeog

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok. Thanks 😄

**Pull request merged**